libreapi/main.go

```diff
@@ -1,12 +1,19 @@
 package main
 
-import "github.com/CamiloHernandez/libreapi/pkg"
+import (
+	"fmt"
+	"github.com/CamiloHernandez/libreapi/pkg"
+	"os"
+)
 
 func main() {
 	certs := libreapi.TLSPaths{
 		CertificatePath: "./certs/libreapi.pem",
 		KeyPath:         "./certs/libreapi.key",
 	}
 
-	panic(libreapi.Start(443, certs))
+	if err := libreapi.Start(443, certs); err != nil {
```

Comment:
I disagree with this change. An unrecoverable fatal error should in fact panic. This code in fact does pretty much the same while being less idiomatic

Reply:
I don't see what you gain panicking on the main function and yes, it does the same without printing the whole stack trace. I agree panic is useful when you want to avoid programmer mistakes and help and give them all the information to have them fix the issue. This case does not provide you anything worth but the stack trace.